server/utils/helpers/filterDateRangeHelper.js

```diff
@@ -0,0 +1,24 @@
+const filterDateRangeHelper = (start, end) => {
+  let startDate; let endDate;
```

Comment:
You could have written `let startDate, endDate;` for brevity.

Reply:
The Eslint configuration we use is against that practice